.circleci/config.yml

```diff
@@ -1,38 +1,36 @@
 ---
 version: 2.1
+
 orbs:
   release-management: salesforce/npm-release-management@4
-jobs:
-  node-latest: &test
-    docker:
-      - image: node:latest
-    working_directory: ~/cli
-    environment:
-      NYC: "yarn exec nyc -- --nycrc-path node_modules/@oclif/nyc-config/.nycrc"
-    steps:
-      - checkout
-      - restore_cache: &restore_cache
-          keys:
-            - v1-yarn-{{checksum ".circleci/config.yml"}}-{{ checksum "yarn.lock"}}
-            - v1-yarn-{{checksum ".circleci/config.yml"}}
-      - run: yarn
-      - run: yarn add -D nyc@13 @oclif/nyc-config@1
-  node-12:
-    <<: *test
-    docker:
-      - image: node:12
-  node-10:
-    <<: *test
-    docker:
-      - image: node:10
 
 workflows:
   version: 2
-  test:
+  test-and-release:
     jobs:
-      - node-latest
-      - node-12
-      - node-10
+      - release-management/test-package:
+          matrix:
+            parameters:
+              os:
+                - linux
+                - windows
+              node_version:
+                - latest
+                - lts
+                - maintenance
+            exclude:
+              - os: windows
+                node_version: lts
+              - os: windows
+                node_version: maintenance
```

Comment:
We don't need these exclusions anymore since the orb can now correctly install different versions of node on Windows based on release tags.